interfaces/builtin/microceph.go

```diff
@@ -32,6 +32,21 @@ const microcephConnectedPlugAppArmor = `
 # Description: allow access to the MicroCeph control socket.
 
 /var/snap/microceph/common/state/control.socket rw,
+
+# Allow bcache devices to be accessed since DM devices may be set up on top of those.
+/dev/bcache[0-9]{,[0-9],[0-9][0-9]} rwk,                   # bcache (up to 1000 devices)
+# Access to individual partitions
+/dev/hd[a-t][0-9]{,[0-9],[0-9][0-9]} rwk,                  # IDE, MFM, RLL
+/dev/sd{,[a-h]}[a-z][0-9]{,[0-9],[0-9][0-9]} rwk,          # SCSI
+/dev/sdi[a-v][0-9]{,[0-9],[0-9][0-9]} rwk,                 # SCSI continued
+/dev/vd[a-z][0-9]{,[0-9],[0-9][0-9]} rwk,                  # virtio
```

Comment:
Should NVMEs be part of that list?

Reply:
Thanks @simondeziel , you're quite right, they probably should